workspaces/default/themes/light-theme/layouts/catalog.html

```diff
@@ -1,7 +1,8 @@
 {% layout = "layouts/_page.html" %}
 
 {% context.tags = {} %}
-{% for _, spec in each(portal.specs) do %}
+{% context.specs = portal.specs_by_tag(page.catalog_tag) %}
```

Comment:
dumb question, where does page.catalog_tag come from?

Reply:
Not dumb question!  catalog_tag can be set in the content file calling this layout, if it is not set it is sent as `nil` and the catalog renders all specs